src/registry/registry.json

```diff
@@ -2884,7 +2915,8 @@
     "accessControlPolicies": "accesscontrolpolicy",
     "integrationHub": "integrationhubsettings",
     "ObjectHierarchyRelationship": "objecthierarchyrelationship",
-    "IndustriesManufacturingSettings": "industriesmanufacturingsettings"
+    "IndustriesManufacturingSettings": "industriesmanufacturingsettings",
```

Comment:
This one: `IndustriesManufacturingSettings` is bad, bad boy, I caused some trouble in the past because of it's extension name.